Lib/test/test_abc.py

```diff
@@ -410,6 +410,13 @@ class C:
             with self.assertRaises(TypeError):
                 issubclass(C(), A)
 
+            # bpo-34441: Check that issubclass() doesn't crash on bogus classes
```

Comment:
Would be nice to add also tests for following cases:

* `__subclasses__` is a callable with wrong signature.
* `__subclasses__` raises an exception explicitly.
* `__subclasses__` returns not a list.
* `__subclasses__` returns a list containing not a type.

Reply:
Thanks, I've added those cases.